content/updates/2020-10/4-Job-and-process-tracker-improvements.md

```diff
@@ -8,25 +8,25 @@ pim_announcement_audience:
 
 # Jobs and process tracker improvements
 
-Check our latest process tracker enhancements: a shortcut for a rapid access to quick exports job details, and a new sorting option in the process tracker grid.
+Check our latest process tracker enhancements: a shortcut for a fast access to quick export job details, and a new sorting option in the process tracker grid.
 
 ## Easy access to the detail of your quick export once launched
 ::: meta-data type="Improvement" features="Productivity" available="mid-October" link-to-doc="../articles/quick-export.html#how"
 :::
 
-When you launch a quick product export from the product grid, you can now easily jump to this task's progress detail in the process tracker. Indeed, you'll find a link to the job detail directly inside the flash message that confirms the quick export launch.
+When you launch a quick product export from the product grid, you can now easily jump to this task's progress detail in the process tracker. Indeed, you'll find a link to the job detail embedded in the flash message that confirms the quick export launch.
 
 ![Flash message with the direct link to the quick export detail](../img/flash-message-with-direct-link-to-the-quick-export-detail.png)
 
 ## Process tracker display enhancements
 ::: meta-data type="New" features="Productivity" available="mid-October" link-to-doc="../articles/monitor-jobs.html#searching-for-a-job"
 :::
 
-Quickly find a launched job thanks to the new `User` filter we introduced in the process tracker grid. It will allow you to either easily find your own jobs or the ones launched by one of your colleagues. Neat!
+Quickly find a launched job thanks to the new `User` filter we introduce in the process tracker grid. It allows you to easily find either your own jobs or the ones launched by one of your colleagues. Neat!
 
 ![User filter](../img/user-filter-in-the-process-tracker.png)
 
-On a side note, you will also notice that you can order the process tracker grid jobs by the user who launched it.
+On a side note, you will also notice that you can organize the process tracker grid jobs by users.
```

Comment:
See our Slack. ;)

Reply:
```suggestion
On a side note, you will also notice that you can sort the process tracker grid jobs by users.
```
@LaureBro, we need your advice here. Is it clear enough from your point of view, if we only mention "users" and not "the user who launched it"?

Reply:
Yes users only is fine for me :)